monitor/internal/pod/controller.go

```diff
@@ -54,6 +54,7 @@ func newReconciler(mgr manager.Manager, handler *config.ProcessorConfig, metadat
 
 		// TODO: might move into configuration
 		handlePUEventTimeout:   5 * time.Second,
+		getterTimeout:          2 * time.Second,
```

Comment:
The above is very short as discussed.

Reply:
and handled through reconciling on errors anyway which is a must at the moment as explained. will change in the phase 3 of the implementation then.